server/remote/bitbucket/bitbucket.go

```diff
@@ -295,6 +295,12 @@ func (c *config) Branches(ctx context.Context, u *model.User, r *model.Repo) ([]
 	return branches, nil
 }
 
+// BranchHead returns the sha of the head (lastest commit) of the specified branch
+func (c *config) BranchHead(ctx context.Context, u *model.User, r *model.Repo, branch string) (string, error) {
+	// TODO(1138): implement #1138
```

Comment:
```suggestion
	// TODO(1138): missing implementation
```

#1138 is a summary ...